### ISSUE TITLE
Vi trenger å endre utmeldingsdato på våre medlemmer

### ISSUE THREAD

**Issue by weblady** [webrtd/website#133]
Vi MÅ få tilgang til å endre utmeldingsdato på medlemmene.
Nå hadde LS en sak med et medlem som fyller 45 i oktober, det betyr at hun kan gå et år til og gå ut 30 juni 2016. Men hun ønsket ikke det, og hvis LS da hadde meldt henne ut så hadde hun forsvunnet fra listen.
Derfor måtte LS nå endre på fødselsdatoen hennes slik at hun fylte år i juni og ikke i oktober for at det skulle bli "riktig".

**Comment by webrtd** [comment]
Det forstår jeg ikke. Er set fordi de skal gå ud et år før?



—
Rasmus Kaae
Mobil: +45 28720413

**Comment by weblady** [comment]
Ja, de ønsker slutte et år før de går ut på dato. Men om vi da melder ut medlemmet, så forsvinner medlemmet helt fra aktive og gamle medlemmer også. Derfor må vi jukse med fødselsdatoen for at medlemmene ikke skal forsvinne.

**Comment by webrtd** [comment]
Ok. 




Det er stadig 'by design' og et fix kræver 1-2 timers programmering på regning. 



—
Rasmus Kaae
Mobil: +45 28720413

**Comment by weblady** [comment]
Ok. Da sjekker jeg med resten av styret om vi ønsker denne endringen før jeg gir tilbakemelding på det. Pris?

**Comment by weblady** [comment]
Pris på dette? 750kr timen som avtalt på tidligere saker vi venter på å få fikset? :)

**Comment by webrtd** [comment]
Hvad skal den korrekte dato være? Som det er pt så udmeldes personer ved udgangen af det klubår de fylder år.

Get Outlook for iOS

On Thu, Jun 30, 2016 at 8:05 AM +0200, "weblady" notifications@github.com wrote:

Pris på dette? 750kr timen som avtalt på tidligere saker vi venter på å få fikset? :)

**Comment by weblady** [comment]
Generelt er det riktig at de skal meldes ut ved utgangen av året som de meldes ut, men om de ønsker å slutte et år før de går ut på dato så blir det litt kluss. Om vi da melder ut medlemmet, så forsvinner medlemmet helt fra aktive og gamle medlemmer også. Derfor må landssekretær jukse med fødselsdatoen for at medlemmene ikke skal forsvinne.

LS hadde en sak med et medlem som fyller 45 i oktober, det betyr at hun kan gå et år til og gå ut 30 juni 2016. Men hun ønsket ikke det, og hvis LS da hadde meldt henne ut så hadde hun forsvunnet fra listen.
Derfor måtte LS nå endre på fødselsdatoen hennes slik at hun fylte år i juni og ikke i oktober for at det skulle bli "riktig".

Er dette forståelig? Jeg har egentlig bare videreformidlet akkurat denne saken fra landssekretær og har ikke opplevd det selv.

**Comment by webrtd** [comment]
Så en løsning kunne være at medlemmer der udmeldes før tid vises under gamle medlemmer?

Get Outlook for iOS

On Thu, Jun 30, 2016 at 4:11 PM +0200, "weblady" notifications@github.com wrote:

Generelt er det riktig at de skal meldes ut ved utgangen av året som de meldes ut, men om de ønsker å slutte et år før de går ut på dato så blir det litt kluss. Om vi da melder ut medlemmet, så forsvinner medlemmet helt fra aktive og gamle medlemmer også. Derfor må landssekretær jukse med fødselsdatoen for at medlemmene ikke skal forsvinne.

LS hadde en sak med et medlem som fyller 45 i oktober, det betyr at hun kan gå et år til og gå ut 30 juni 2016. Men hun ønsket ikke det, og hvis LS da hadde meldt henne ut så hadde hun forsvunnet fra listen.

Derfor måtte LS nå endre på fødselsdatoen hennes slik at hun fylte år i juni og ikke i oktober for at det skulle bli "riktig".

Er dette forståelig? Jeg har egentlig bare videreformidlet akkurat denne saken fra landssekretær og har ikke opplevd det selv.

**Comment by weblady** [comment]
Ja, det hadde vært en bra løsning :)

Den tor, juni 30, 2016, 16:23 skrev webrtd:

> Så en løsning kunne være at medlemmer der udmeldes før tid vises under
> gamle medlemmer?
> 
> Get Outlook for iOS
> 
> On Thu, Jun 30, 2016 at 4:11 PM +0200, "weblady"
> notifications@github.com wrote:
> 
> Generelt er det riktig at de skal meldes ut ved utgangen av året som de
> meldes ut, men om de ønsker å slutte et år før de går ut på dato så blir
> det litt kluss. Om vi da melder ut medlemmet, så forsvinner medlemmet helt
> fra aktive og gamle medlemmer også. Derfor må landssekretær jukse med
> fødselsdatoen for at medlemmene ikke skal forsvinne.
> 
> LS hadde en sak med et medlem som fyller 45 i oktober, det betyr at hun
> kan gå et år til og gå ut 30 juni 2016. Men hun ønsket ikke det, og hvis
> LS da hadde meldt henne ut så hadde hun forsvunnet fra listen.
> 
> Derfor måtte LS nå endre på fødselsdatoen hennes slik at hun fylte år i
> juni og ikke i oktober for at det skulle bli "riktig".
> 
> Er dette forståelig? Jeg har egentlig bare videreformidlet akkurat denne
> saken fra landssekretær og har ikke opplevd det selv.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub, or mute the thread.
> 
> ---
> 
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/webrtd/website/issues/133#issuecomment-229673642

Med vennlig hilsen

Monica Gjelsten
Weblady 2016/2018
Ladies Circle Norway

**Comment by webrtd** [comment]
Ok, det kan wordpress udgaven ...

Med venlig hilsen / Best regards

_Rasmus Kaae_
National WEB Round Table Denmark http://www.rtd.dk | RT111 Favrskov
http://www.rt111.dk/

2016-07-01 21:46 GMT+02:00 weblady notifications@github.com:

> Ja, det hadde vært en bra løsning :)
> 
> Den tor, juni 30, 2016, 16:23 skrev webrtd:
> 
> > Så en løsning kunne være at medlemmer der udmeldes før tid vises under
> > gamle medlemmer?
> > 
> > Get Outlook for iOS
> > 
> > On Thu, Jun 30, 2016 at 4:11 PM +0200, "weblady"
